.github/workflows/dashboard-testnet.yml

```diff
@@ -0,0 +1,93 @@
+name: Token Dashboard / Testnet
+
+#TODO: extend the conditions once workflow gets tested together with other workflows
+on:
+  push:
+    branches:
+      # TODO: Run on master after we're fully migrated from Circle CI
+      # - master
+      - "rfc-18/**"
+    paths:
+      - "solidity/dashboard/**"
+      - ".github/workflows/dashboard-testnet.yml"
+  pull_request:
+    branches:
+      # TODO: Run on all branches after we're fully migrated from Circle CI
+      - "rfc-18/**"
```

Comment:
Are we ready to remove all parts of the CircleCI configuration related to the token dashboard? Can we switch to GH Actions only?

Reply:
Yes, done in 83fd49a793852688ceae109af4cccefb70dcac00 and c420cde8966f03aa092927d581c65b3339892270.